SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+OWASP CRS has two major types of releases, Major releases (2.x.x, 3.x.x, 4.x.x) 
```

Comment:
can we remove the first major? Too many major.

Reply:
Also, can we use real life examples? e.g. 4.x.x doesn't exist yet.

Reply:
Yes, I'd leave the 2.x.x and the 4.x.x away and just define 3.1.x and 3.2.x as major releases.